test/NinjaTools.FluentMockServer.API.Tests/Services/LogServiceTests.cs

```diff
@@ -0,0 +1,122 @@
+using System.Linq;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using NinjaTools.FluentMockServer.API.Helper;
+using NinjaTools.FluentMockServer.API.Models;
+using NinjaTools.FluentMockServer.API.Models.Logging;
+using NinjaTools.FluentMockServer.API.Services;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Services
+{
+    public class LogServiceTests : XUnitTestBase<LogServiceTests>
+    {
+        /// <inheritdoc />
+        public LogServiceTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private ILogService CreateSubject(GenerateId idGenerator = null)
+        {
+            if (idGenerator != null)
+            {
+                return new LogService(CreateLogger<LogService>(), new LogFactory(idGenerator));
+            }
+
+            return new LogService(CreateLogger<LogService>());
+        }
+
+        public const string Id = "1234";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)